src/Bridge/Doctrine/Repositories/Interfaces/WebhookResponseRepositoryInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Webhooks\Bridge\Doctrine\Repositories\Interfaces;
+
+interface WebhookResponseRepositoryInterface
```

Comment:
Any repository interface should extend from the base repository interface imo

Reply:
That's what I thought initially but then `WebhookRequestRepositoryInterface` wasn't extending base repository interface.

Reply:
Resolution:

Extend base repository interface in request and response repository